society-of-biblical-literature-fullnote-bibliography.csl

```diff
@@ -313,20 +332,76 @@
     </choose>
   </macro>
   <macro name="collection-title">
-    <text variable="collection-title" form="short"/>
-    <text variable="collection-number" prefix=" "/>
+    <!--TODO: Macro is identical to collection-title-note. Seperate macros not needed?-->
```

Comment:
If the macros are identical, we'd not want to duplicated them.

Reply:
This comment is no longer relevant as I had to add some volume handling to the collection-title macro that is different from the collection-title-note macro. I'll remove comments which are no longer relevant via a new commit below.

Reply:
Fixed with commit f3269e6.